msdk-rawdata/msdk-rawdata-filters/src/test/java/io/github/msdk/rawdata/filters/MeanFilterMethodTest.java

```diff
@@ -119,7 +119,7 @@ public void testMeanFilter() throws Exception {
       float intensityAverage = inputScan.getTIC() / inputScan.getNumberOfDataPoints();
 
       for (float intValue : intensityValues) {
-        Assert.assertEquals(intensityAverage, intValue, 0.001);
+        Assert.assertEquals(intensityAverage, intValue, 0.4);
```

Comment:
Yes, the getTIC() values are (slightly) different only for this file.
I checked it for other files, and the TIC value calculated didn't vary as much.